generator/log/values.go

```diff
@@ -119,6 +119,7 @@ func ForAllClusters(namespace, svc model.LabelValue, cb func(model.LabelSet, pus
 				"__stream_shard__": model.LabelValue(shards[clusterInt%len(shards)]),
 				"namespace":        namespace,
 				"service_name":     svc,
+				"file":             "C:\\Grafana\\logs\\" + namespace + ".txt",
```

Comment:
@svennergr this is lovely for e2e tests and internal testing, but maybe not quite as great on play since it makes the data more nonsensical. 

Is it worth adding a command line flag to gate this label so we can keep it out of play?

Reply:
Created: https://github.com/grafana/explore-logs/issues/1049